### PR TITLE
sane defaults for experiment caffe isolation

### PR DESCRIPTION
**Pull request by ppalucki**
Fixes issue "run caffe without core isolation by default" - makes default results more sensible (and with align with presented results)

Summary of changes:
- change LLC isolation for caffe to false by default

Testing done:
- yes

### CODE REVIEW

**Comment by ppalucki** [comment]
run integration tests

File: experiments/memcached-cat/main.go [hunk 1]
```diff
@@ -143,6 +144,9 @@ func main() {
 		aggressors = append(aggressors, "")
 	}
 
+	rdtSession, err := rdt.NewSessionLauncherDefault()
```
Comment (skonefal) [inline experiments/memcached-cat/main.go]: Not yet.

**Comment by ppalucki** [comment]
run integration tests

**Comment by ppalucki** [comment]
run integration tests

**Pull request merged**